apps/web/src/blocks/Layout/index.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
+
+import React, { PropsWithChildren } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { css, styled } from 'styled-components';
+
+import Button from 'ui/components/Button';
+
+import { Layout as LayoutType, Media } from '@web/payload/payload-types';
+import isExpandedDoc from '../../lib/isExpandedDoc';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  ${({
+    theme: {
+      mq: { lg }
+    }
+  }) => lg`
+    flex-direction: row;
+    gap: unset;
+  `}
+`;
+
+const MenuLinksContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 28px;
+`;
+
+export const MenuLink = styled.p`
+  ${({ theme: { colors } }) => css`
+    color: ${colors.white};
+  `}
+  && {
+    font-size: 17px;
+    font-style: normal;
+    text-decoration: none;
+    font-weight: 400;
+    line-height: normal;
+    letter-spacing: -0.17px;
+    margin: 0;
+    padding: 0;
+  }
+`;
+
+function MenuLinks({ links }: { links: LayoutType['headerItems'] }) {
+  return (
+    <MenuLinksContainer>
```

Comment:
Do we need this container for the array since it's already wrapped by the header/footer?

Reply:
This was added to prevent parent flex styles to be applied to the child menu link components. The container allows me to use `space-between` on the parent and then individual flex styles within the menu links container.